plugins/woocommerce/tests/php/includes/class-wc-review-order-after-cart-item-meta-hook-test.php

```
@@ -0,0 +1,209 @@
+
+	/**
+	 * Test that hook works with variable products.
+	 */
+	public function test_hook_works_with_variable_products() {
+		// Create a variable product
+		$variable_product = WC_Helper_Product::create_variation_product();
+		
+		// Get a variation
+		$variations = $variable_product->get_available_variations();
+		$variation = reset( $variations );
+		$variation_product = wc_get_product( $variation['variation_id'] );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$variation_product'. (undefined)

(UnusedLocalVariable)